src/plugins/vis_type_timelion/server/usage_collector/register_timelion_collector.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { first } from 'rxjs/operators';
+import { getStats, TimelionUsage } from './get_usage_collector';
+import { ConfigObservable } from '../types';
+
+export function registerTimelionUsageCollector(
+  collectorSet: UsageCollectionSetup,
+  config: ConfigObservable
+) {
+  const collector = collectorSet.makeUsageCollector<TimelionUsage | undefined>({
+    type: 'vis_type_timelion',
+    isReady: () => true,
+    schema: {
+      timelion_use_scripted_fields_90_days_total: { type: 'long' },
```

Comment:
NIT: Do you mind adding the description of the field under `_meta.description`? i.e.: 

```suggestion
      timelion_use_scripted_fields_90_days_total: { 
        type: 'long',
        _meta: { description: 'Number of timelion visualizations using scripted fields created/updated in the last 90 days' }
      },
```

Reply:
I haven't seen this property in other collectors in visualizations. If you would like I can add it.

Reply:
@VladLasitsa yes, it's a recent addition (#89685) and we're trying to encourage devs to document the fields they report. This will be useful down the line when our analysts want to understand what this info is for.